app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1519,8 +1530,10 @@ public boolean onPrepareOptionsMenu(Menu menu) {
     toggleActionItemsConfig();
     this.menu = menu;
 
-    if (getCurrentWebView().getUrl() == null ||
-        getCurrentWebView().getUrl().equals(HOME_URL)) {
+    // check for webViewList's size so new tab doesn't open on clicking of three dot menu
+    // issue "https://github.com/kiwix/kiwix-android/issues/995#issue-411106387"
+    if (webViewList.size() != 0 && (getCurrentWebView().getUrl() == null ||
```

Comment:
@prajurock removed all the comments you mentioned, and i think check for "webViewList" size is necessary as this if statement was calling "getCurrentWebView()" method when clicked on menu button and was adding new tab.
i saw #1026, 1500ms is small time but still if user click menu button in that time it will open one new tab and one another after 1500ms.
so "webViewList.size()" check is just to prevent opening of new tab on clicking menu button.